src/Stormpath/Cache/MemcachedCacheManager.php

```diff
@@ -0,0 +1,28 @@
+<?php namespace Stormpath\Cache;
+
+
+
+class MemcachedCacheManager implements CacheManager {
+
+
+    public function __construct($options)
+    {
+        $this->options = $options;
+    }
+
+    /**
+     * @return MemcachedCache
+     * Privide region
+     * cache per region
+     * Singleton for the regions cache
+     */
+    public function getCache()
+    {
+
+        return new MemcachedCache($this->options);
```

Comment:
What about getting the Cache instance from the cache String key (i.e.: from the region)?